thumbor/doctor.py

```diff
@@ -161,18 +161,19 @@ def check_extensibility_modules(cfg):
             modules = [modules]
 
         for module in modules:
-            try:
-                import_module(module)
-                print_success(f"{CHECK} {module}")
-            except ImportError as error:
-                print_error(f"{CROSS} {module} - {error_message}")
-                errors.append(
-                    format_error(
-                        module,
-                        str(error),
-                        error_message,
+            if module is not None:
+                try:
+                    import_module(module)
+                    print_success(f"{CHECK} {module}")
+                except ImportError as error:
+                    print_error(f"{CROSS} {module} - {error_message}")
```

Comment:
Would you consider decreasing the level of indentation of the code and continuing the loop earlier?

```suggestion
            if not module:
                continue
            try:
                import_module(module)
                print_success(f"{CHECK} {module}")
            except ImportError as error:
                print_error(f"{CROSS} {module} - {error_message}")
                # ...
```

That would make the code read more easily while behaving the same.